### PR TITLE
Add hasHeader() method to Message

### LINKED ISSUE #224: Add Message::hasHeader()
Currently we have `getHeader()` and `setHeader()` and `removeHeader()`, but no `hasHeader()`.

### PR DESCRIPTION
**Pull request by ekinhbayar**
Fixes #224 with TrailersTest testing this method.

### CODE REVIEW

**Comment by kelunik** [comment]
Please update the code style (you can run `./vendor/bin/php-cs-fixer fix`) and rebase onto the current master so the tests run fine. :+1:

File: test/TrailersTest.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
```
Comment (kelunik) [inline test/TrailersTest.php]: We don't use strict types. :)

Comment (ekinhbayar) [reply test/TrailersTest.php]: ^^ right... Fixed.

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**